ShareExtension/ShareViewController.swift

```diff
@@ -0,0 +1,76 @@
+import Cocoa
+
+final class ShareViewController: NSViewController {
+	override var nibName: NSNib.Name? { NSNib.Name("ShareViewController") }
+
+	override func loadView() {
```

Comment:
There's a SwiftLint violation here. I think it's a false-positive, so you need to add an ignore comment.

Reply:
Done